lib/hammer_cli_foreman_remote_execution/job_invocation.rb

```diff
@@ -52,7 +54,18 @@ class InfoCommand < HammerCLIForeman::InfoCommand
       end
 
       def extend_data(invocation)
-        JobInvocation.extend_data(invocation)
+        invocation = JobInvocation.extend_data(invocation)
+        if option_inputs?
+          inputs = [] 
+          invocation['template_invocations'].each do |template|
+            input_values = template['template_invocation_input_values']
```

Comment:
The inputs are per-host. Could we maybe pair them with hosts when listing them so the users don't have to guess which inputs go to which host?

Reply:
I'm not sure how can I get the host per input? We don't do it in the UI and I can't see it in the API

Reply:
We do it in the UI for first 20 hosts, it is in job invocation details > preview templates and clicking one of the entries there. We currently don't  pass all the needed information to link template invocations to hosts in the api so you would have to include the missing IDs so you could pair hosts with template invocations in hammer. Alternatively we could reshape that data the api gives, but that would probably be considered breaking change.

Reply:
Added https://github.com/theforeman/foreman_remote_execution/pull/538 for it